lib/fpm/package/rpm.rb

```diff
@@ -301,6 +301,11 @@ def converted_from(origin)
         end
       end.flatten
     end
+   
+    # Erase troublesome release candidate declarations.
+    self.dependencies = self.dependencies.collect do |dep|
+      dep.gsub(/.rc\d*/, '')
```

Comment:
Yeah. I figured release candidate was the worst offender and was the one the tripped me up in particular.